package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "ng lint",
     "e2e": "ng e2e",
     "ver": "node -v",
-    "build": "npm run ng-build & copyfiles CNAME docs"
+    "build": "npm run ng-build & cp CNAME docs/"
```

Comment:
@jdjuan  If you want to use copyfiles just change `copyfiles CNAME docs` for `./node_modules/.bin/copyfiles CNAME docs`